doc/Developing.md

```diff
@@ -127,7 +127,7 @@ sbt keeps the cache of downloaded jar files in `${HOME}/.ivy2/cache`. For exampl
 - Update release notes and README.md
 - Make sure the version number in `src/main/resources/application.conf` is correct. It is used
 when building the release.
-- Merge onto master branch, and make sure [travis tests](https://travis-ci.org/dnanexus/dxWDL) pass
+- Merge onto master branch, and make sure all Github Actions pass
```

Comment:
- add a link https://github.com/dnanexus/dxWDL/actions
- should say "all Github Actions tests"

Reply:
fixed